csp_billing_adapter_local/plugin.py

```diff
@@ -132,6 +133,7 @@ def get_usage_data(config: Config):
     :param config: The application configuration dictionary
     :return: Return a dict with the current usage report
     """
+    now = datetime.datetime.now(datetime.timezone.utc).isoformat()
```

Comment:
Please leverage `date_to_string(get_now())` from https://github.com/SUSE-Enceladus/csp-billing-adapter/blob/main/csp_billing_adapter/utils.py. This ensures all timestamps match and if format changes it only affects one spot in code.